1.alert message.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title> alert message</title>   
```

Comment:
Remove the space before alert and make it camel cased.

Reply:
done.